### PR TITLE
add custom 404 page

### PR DESCRIPTION
**Pull request by dkhgh**
not that pretty, but better than nothing :+1: 

screen shot 2016-01-30 at 3 17 32 pm

### CODE REVIEW

**Comment by dkhgh** [comment]
* should work with s3 (I've set the buckets up to treat 404.html as error document)
* found out the hard way that keys in our translation files can not be numbers :cactus:

File: source/localizable/404.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Pricing
```
Comment (fatboypunk) [inline source/localizable/404.html.erb]: Title is pricing?

Comment (sn3p) [inline source/localizable/404.html.erb]: Noticed that as well, fixed in commit below ;)

**Comment by sn3p** [comment]
Fixes https://github.com/DefactoSoftware/LearningSpaces-Landing/issues/60

* disabled `directory_index`
* 404 page titles
* excludes 404.html from sitemap

**Pull request merged**